apps/staking/components/ScreenContainer.tsx

```diff
@@ -1,5 +1,16 @@
 import type { ReactNode } from 'react';
+import { getRemoteFeatureFlag } from '@/lib/feature-flags-server';
+import { REMOTE_FEATURE_FLAG } from '@/lib/feature-flags';
+import Maintenance from '@/components/Maintenance';
+
+export default async function ScreenContainer({ children }: { children: ReactNode }) {
+  const { enabled: isAppMaintenanceModeEnabled } = await getRemoteFeatureFlag(
```

Comment:
What is the difference betwen app maintence and maintenance?

Reply:
app maintenance is for the app, so still info content and header/footer, just no app (no mystakes open nodes, or registration). Maintenance is the entire site

Reply:
Nice resolving